Dockerfile-linux.template

```diff
@@ -173,8 +173,10 @@ RUN set -eux; \
 		EXTRA_CFLAGS="-DTHREAD_STACK_SIZE=0x100000" \
 {{ ) else "" end -}}
 {{ if is_slim or is_alpine then ( -}}
-		LDFLAGS="-Wl,--strip-all" \
-{{ ) else "" end -}}
+		LDFLAGS="-Wl,--strip-all,-rpath='$\$ORIGIN/../lib'" \
```

Comment:
```suggestion
# \$ because of the double quotes in the shell to prevent interpolation
# $$ for make to not interpret the $O
# " because it needed the ' around the path, but I didn't want ='-Wl,-rpath='"'"'$$ORIGIN/../lib'"'"
		LDFLAGS="-Wl,{{ if is_slim or is_alpine then "--strip-all," else "" end }}-rpath='\$\$ORIGIN/../lib'" \
```

(and then dropping the outer `if`? :eyes:)